CMakeLists.txt

```diff
@@ -844,6 +844,29 @@ configure_file ( fluidsynth.pc.in
 install ( FILES ${CMAKE_BINARY_DIR}/fluidsynth.pc
 	DESTINATION ${LIB_INSTALL_DIR}/pkgconfig )
 
+# Exported targets for cmake: find_package(FluidSynth)
+# when installed, use CMAKE_PREFIX_PATH=fluidsynth-prefix;...
+# to use the build directory directly set the FluidSynth_DIR variable instead.
+
+# targets in the build directory
+export(EXPORT FluidSynthTargets
+   FILE "${CMAKE_CURRENT_BINARY_DIR}/FluidSynthTargets.cmake"
+   NAMESPACE FluidSynth::
+)
+
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+    FluidSynthConfigVersion.cmake
+    VERSION ${LIB_VERSION_INFO}
```

Comment:
I'm not sure about LIB_VERSION here. The docs say

> [This] is the version number of the project to be installed.

So, I would tend to go for `FLUIDSYNTH_VERSION` instead.

Reply:
The variable `FLUIDSYNTH_VERSION` can't be used for this, I've used the `VERSION` variable instead. Also, for `COMPATIBILITY`, the value `SameMinorVersion` is a much better choice than `SameMajorVersion` for FluidSynth, in case that this patch would be backported to old releases.